### ISSUE TITLE
Get Motivated

### ISSUE THREAD

**Issue by farosato** [farosato/fabiorosatodotcom#6]
---
slug: get-motivated
---
Motivation is a dangerous mistress.

You shouldn't (just) rely on it, but lacking it completely makes for a very sour and dull life.

So here’s what I've found to work pretty well for me after reading more self-help and productivity articles and books than I'm comfortable admitting.

**1. Define your goals properly**

Frame it with the (happy) END in mind. Your goal should excite you!

"Get a Hollywood hero physique" over "Go to the gym"

Doing something cause you want to is more exciting than doing it cause you need to.

**2. Set an ambitious but attainable target**

Think [Goldilocks](https://en.wikipedia.org/wiki/Goldilocks_principle).

Make it challenging enough that you don't slack off, but accessible enough that you don't get paralyzed.

Use **specific** numbers.

"Lift weights 3x/week for 45 minutes" over "Workout more".

**3. Monitor your progress**

Making progress is addicting. So make sure you're tracking it!

On the low days, use the [sunk cost fallacy](https://thedecisionlab.com/biases/the-sunk-cost-fallacy) to push you forward.
Don't break the chain!

(You’re almost done with this post, Fabio. Don’t quit now!)

**4. Survive the middle slum**

Motivation is highest at the beginning and end of a project.

_The middle is dangerous territory._

To beat it, once you're past the midpoint flip from reviewing progress made to _monitoring the shrinking amount of work left_. 

(Just 1 more bullet, Fabio!)

**5. Think about your future self**

It may be ok to drop it _today_.
You just don't feel like it.

But ask yourself whether it's ok to do that for the rest of your life.

**We are what we repatedly do.**
So think about your future.
Think about who you want to be.

**And be that person NOW.**

> TLDR
> 1. Define your goals properly, happy end in mind.
> 2. Set an ambitious but doable specific target.
> 3. Track your progress.
> 4. Survive the middle slum.
> 5. Think about your future self.

Now go do your magic.
